docs/headless-api.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Rojo Plugin APIs
+sidebar_position: 7
+---
+
+The Rojo Studio plugin exposes a headless API into `_G`, which is very valuable to the Rojo ecosystem. This allows you to make companion plugins that interact with Rojo!
+
+### Accessing the API
+```Lua
+local Rojo = _G.Rojo
+```
+Plugins and command bar all share the same _G which allows us to put our interface in. Using the key `"Rojo"` is easy to remember and follows branding.
```

Comment:
Use backticks for all occurrences of `_G`

Reply:
Good catch!